man/ocaml.1

```diff
@@ -165,7 +165,12 @@ use it once before publishing source code.
 .B \-rectypes
 Allow arbitrary recursive types during type-checking.  By default,
 only recursive types where the recursion goes through an object type
-are supported.
+are supported. Note that once you have created an interface using this
+flag, you must use it again for all dependencies.
```

Comment:
The REPL does not really create interface that are reused by other users. It seems better to keep this note out of this specific description.